README.md

```diff
@@ -2,7 +2,11 @@
 [![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/pipeline-input-step.svg)](https://plugins.jenkins.io/pipeline-input-step)
 [![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/pipeline-input-step.svg?color=blue)](https://plugins.jenkins.io/pipeline-input-step)
 
-Adds the Pipeline `input` step to wait for human input or approval.
+Adds the Pipeline `input` step to wait for human input or approval. 
+
+For further understanding,check the [documentation](https://plugins.jenkins.io) for pipeline-input-step plugin.  The  [Pipeline Syntax Snippet Generator](https://www.jenkins.io/redirect/pipeline-snippet-generator) guides the user to select checkout options.
+
+
 
 ## Version History
-Please refer to [the changelog](CHANGELOG.md)
+Please refer to [the changelog](CHANGELOG.md) and SCM [`checkout` step](https://www.jenkins.io/doc/pipeline/steps/workflow-scm-step/).
```

Comment:
I recommend that this section be left as it was previously, a reference to the version history, rather than mixing a reference to something that is not related to version history.
```suggestion
Please refer to [the changelog](CHANGELOG.md)
```